chart/templates/preflight.yaml

```diff
@@ -15,14 +15,7 @@ stringData:
       namespace: {{ .Release.Namespace | quote }}
     spec:
       collectors:
-        - postgres:
-            collectorName: workspace-db
-            uri: postgresql://{{ include "carto.postgresql.adminUser" . | trimAll "\"" }}:{{ .Values.externalPostgresql.adminPassword | trimAll "\"" }}@{{ include "carto.postgresql.host" . | trimAll "\"" }}:{{ include "carto.postgresql.port" . | trimAll "\"" }}/{{ include "carto.postgresql.adminDatabase" . | trimAll "\"" }}
-        {{- if not .Values.internalRedis.enabled }}
-        - redis:
-            collectorName: redis
-            uri: redis://:{{ .Values.externalRedis.password | trimAll "\"" }}@{{ include "carto.redis.host" . | trimAll "\"" }}:{{ include "carto.redis.port" . | trimAll "\"" }}
-        {{- end }}
+    {{- include "carto.replicated.commonChecks.collectors" . | indent 6 }}
         - registryImages:
```

Comment:
I thought it has more sense do this check before to install or do a upgrade, for this I put them in the preflights. Maybe could be useful to have this info also in the support bundle in case a customer do a upgrade without see the preflights

Reply:
That's my idea, we should have that info in the support bundle.